src/components/CategoriesNav/index.js

```diff
@@ -76,6 +76,9 @@ const CategoriesNav = ({ emojiListRef }) => {
           if (groupVisibility[group] === false) {
             return null;
           }
+          if (group === 'custom' && customGroups === {}) {
+            return null;
+          }
```

Comment:
I am not sure relying on a hardcoded name ("custom") is the best way to go about this. You already have the actual names of the groups in the `customGroups` object. You can easily check that the name is the same group name from the customGroups object

Reply:
fixed on commit 3223e3a4be461f3624313a30b4bdf93731f50eaa